src/views/catalog/templatescatalog/hooks/useIsTemplateSupportsQuickCreate.ts

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+export const ProcessedTemplatesModel: K8sModel = {
```

Comment:
@yaacov do we need this model?

Reply:
do the sdk expose any models ?

Reply:
the sdk doesn't expose models

Reply:
than we will add it to the API repo :+1: